library/src/main/java/fr/tvbarthel/cheerleader/library/player/NotificationManager.java

```diff
@@ -122,6 +127,11 @@
     private NotificationConfig mNotificationConfig;
 
     /**
+     * A {@link Runnable} to load the artwork with some delay.
+     */
+    private Runnable loadArtworkRunnable;
```

Comment:
unfortunatelly, Hungarian notation is used for this project :x please tweak this to keep the current convention.